sql/driver/driver_test.go

```diff
@@ -31,7 +32,7 @@ var isError = testutils.IsError
 
 func setup(t *testing.T) (*server.TestServer, *sql.DB) {
 	s := server.StartTestServer(nil)
-	db, err := sql.Open("cockroach", "https://root@"+s.ServingAddr()+"?certs=test_certs")
+	db, err := sql.Open("cockroach", "https://"+security.RootUser+"@"+s.ServingAddr()+"?certs=test_certs")
```

Comment:
We do not, and while I agree with you when it comes to links, we use it everywhere throughout the code. Changing a named constant is much much simpler than digging around for "root". I was actually going to push this further by removing storage.UserRoot and replacing all instances of plain-text "root".